src/app/data-planes/components/DataplanePolicies.vue

```diff
@@ -221,10 +224,9 @@ function getPolicyTypeEntries(sidecarDataplanes: SidecarDataplane[]): PolicyType
       }
 
       const policyTypeEntry = policyTypeEntriesByType.get(policyTypeName) as PolicyTypeEntry
-      const policyType = store.state.policyTypesByName[policyTypeName] as PolicyType
 
       for (const policy of policies) {
-        const connections = getPolicyTypeEntryConnections(policy, policyType, sidecarDataplane, destinationTags, name)
+        const connections = getPolicyTypeEntryConnections(policy, sidecarDataplane, destinationTags, name)
```

Comment:
Why change this to now rely on global state instead?

Reply:
Especially as in the future this function won't be called as part of a watch and will just be called directly from the template and passed down the data from Datasources

Reply:
> Especially as in the future this function won't be called as part of a watch and will just be called directly from the template and passed down the data from Datasources

Ah, this comment just appeared.

So, you would like to make this a pure function so it can be moved more easily?

Reply:
Yeah. Moreover, I don't see how changing this makes it any better.

Reply:
I’ve changed this back.